src/main/java/io/papermc/hangar/service/project/ProjectService.java

```diff
@@ -53,16 +54,18 @@
     private final HangarDao<ProjectApiDao> projectApiDao;
     private final UserService userService;
     private final FlagService flagService;
+    private final Jdbi jdbi;
 
     @Autowired
-    public ProjectService(HangarConfig hangarConfig, HangarDao<ProjectDao> projectDao, HangarDao<UserDao> userDao, HangarDao<VisibilityDao> visibilityDao, HangarDao<ProjectApiDao> projectApiDao, UserService userService, FlagService flagService) {
+    public ProjectService(HangarConfig hangarConfig, HangarDao<ProjectDao> projectDao, HangarDao<UserDao> userDao, HangarDao<VisibilityDao> visibilityDao, HangarDao<ProjectApiDao> projectApiDao, UserService userService, FlagService flagService, Jdbi jdbi) {
```

Comment:
not really sure why JDBI is injected here?

Reply:
ah, I forgot to remove it